### PR TITLE
feat: support filter sets

### LINKED ISSUE #29: Filtered Policy Doesn't support a set of filter rules
Comparing how filtered policies work for something like the gorm go adapter https://github.com/casbin/gorm-adapter/blob/115228287704942b18ceb28d0b5ae5860025bee7/adapter.go#L401 and this adapter, the current implementation doesn't support filter sets. Currently it just takes whatever the first filter is and just runs with that.

### PR DESCRIPTION
**Pull request by wakemaster39**
resolves #29

### CODE REVIEW

**Comment by wakemaster39** [comment]
Just a question, if there is a reason to keep the length checks and instead just use truthiness?

**Comment by hsluoyz** [comment]
@techoner @Zxilly plz review.

File: casbin_sqlalchemy_adapter/adapter.py [hunk 1]
```diff
@@ -23,7 +25,7 @@ def __str__(self):
             if v is None:
                 break
             arr.append(v)
-        return ', '.join(arr)
+        return ", ".join(arr)
```
Comment (Zxilly) [inline casbin_sqlalchemy_adapter/adapter.py:28]: Maybe there is no need to change these

**Review by Zxilly — CHANGES_REQUESTED** [review]

File: casbin_sqlalchemy_adapter/adapter.py (hunk 1 above)
Comment (Zxilly) [reply casbin_sqlalchemy_adapter/adapter.py:28]: I think the other part look good to me.

**Review by Zxilly — APPROVED** [review]

**Comment by hsluoyz** [comment]
@Zxilly plz make a new release.

**Comment by hsluoyz** [comment]
@techoner can you help make a new release?

https://pypi.org/project/casbin-sqlalchemy-adapter/

**Comment by wakemaster39** [comment]
Sweet, thanks @hsluoyz. Sorry black just auto runs on all my code, I didn't even notice the change in quotes

**Pull request merged**